pkg/bgpv1/api/printers.go

```diff
@@ -190,3 +210,103 @@ func formatPathActions(stmt *models.BgpRoutePolicyStatement) []string {
 	}
 	return res
 }
+func printCapability(c *models.BgpCapability) string {
+	bin, err := base64.StdEncoding.DecodeString(c.Capability)
+	if err != nil {
+		return fmt.Sprintf("Could not decode from base64 to bytes %s: %s", c.Capability, err)
+	}
+	capability, err := bgppacket.DecodeCapability(bin)
+	if err != nil {
+		return fmt.Sprintf("Could not decode from bytes to capability %s: %s", c.Capability, err)
+	}
+	switch capability.Code() {
```

Comment:
@liyihuang I think you could have a cap func registry vs using inline functions. 
You could then just indexed on the registry to pull the associated func to produce the output. 
Not only this will clean up the switch stmt, you can unit test each resolvers to ensure they produce the correct output.

Reply:
Thanks. I think I shouldn't copy too much code from gobgp implementation. I will address that in my next commit